source/base/parameter_handler.cc

```diff
@@ -2386,7 +2386,7 @@ ParameterHandler::recursively_print_parameters (const std::vector<std::string> &
             // the documentation, and then the actual entry; break the
             // documentation into readable chunks such that the whole
             // thing is at most 78 characters wide
-            if ((!(style & 128)) &&
+            if ((style == Text) &&
```

Comment:
This part of the change is inside a block that can only be reached if `style` is either `Text` (==1) or `ShortText` (==193). So `!(style & 128)` is equal to `!(style == ShortText)` which is equal to `style == Text`. So I think the change is correct.